content/data/bloss2018.yaml

```diff
@@ -19,7 +19,7 @@ data:
         path: "bloss/bloss18/image"
         cutout_x: 5750
         cutout_y: 3750
-        cutout_z: 714
```

Comment:
The z width of this image is only 318 (https://viz.neurodata.io/#!%7B%22dimensions%22:%7B%22x%22:%5B3.890000104904175e-9%2C%22m%22%5D%2C%22y%22:%5B3.890000104904175e-9%2C%22m%22%5D%2C%22z%22:%5B5.0000000000000004e-8%2C%22m%22%5D%7D%2C%22position%22:%5B42446.34765625%2C53290.6640625%2C160%5D%2C%22crossSectionScale%22:153.69959261184266%2C%22projectionScale%22:512.1075158713335%2C%22layers%22:%5B%7B%22type%22:%22image%22%2C%22source%22:%7B%22url%22:%22precomputed://https://open-neurodata.s3.amazonaws.com/bloss/bloss18/image%22%7D%2C%22blend%22:%22default%22%2C%22name%22:%22bloss18%22%7D%5D%2C%22layout%22:%22xy%22%7D), so this is invalid